website/i18n/zh/docusaurus-plugin-content-blog/2022/03/01/apisix-integration-public-api-plugin.md

```diff
@@ -0,0 +1,249 @@
+---
+title: "public API 处理能力再增强，API 网关 Apache APISIX 支持 `public-api` 插件"
+authors:
+  - name: "白泽平"
+    title: "Author"
+    url: "https://github.com/bzp2010"
+    image_url: "https://avatars.githubusercontent.com/u/8078418?v=4"
+  - name: "韩飞"
+    title: "Technical Writer"
+    url: "https://github.com/hf400159"
+    image_url: "https://avatars.githubusercontent.com/u/97138894?v=4"
+keywords: 
+- Apache APISIX
+- API Gateway
+- Security
+- Ecosystem
+description: 本文为您介绍了 API 网关 Apache APISIX `public-api` 插件的原理及使用方法。
+tags: [Technology,Ecosystem,Security]
```

Comment:
The `Security` tag is for other usages, please remove it.
```suggestion
tags: [Technology,Ecosystem]
```

Reply:
I also think it is necessary to add spaces after the comma to ensure clear formatting.

Reply:
> I also think it is necessary to add spaces after the comma to ensure clear formatting.

Spaces do not really affect the formatting. We have been using no spaces since August 😂